packages/cli/templates/default-backend-plugin/src/alpha.ts.hbs

```diff
@@ -0,0 +1 @@
+export { {{id}}Plugin as default } from './plugin';
```

Comment:
It's because the backstage feature factory const is defined as `{{id}}Plugin`
https://github.com/janus-idp/backstage-plugins/pull/1132/files/642806e145460ccdaa2d8ede0cde55e4364c82e8#diff-a28d326d372a48db04913b26fc81539025193bec18b8f0a908bd97316cbc989fR11

Reply:
oh wait I misunderstood, I didn't notice that a LF was added.

Reply:
It also doesn't appear in the file when I view it on the detached branch on github.
https://github.com/janus-idp/backstage-plugins/blob/642806e145460ccdaa2d8ede0cde55e4364c82e8/packages/cli/templates/default-backend-plugin/src/alpha.ts.hbs

Reply:
I don't see any issues when running `yarn new --select backend-plugin` so I think this can be safely ignored?

Reply:
Sure